### DISCUSSION TITLE
Change UI font-size Atom/material

### DISCUSSION THREAD

**Discussion by johfa800** [atom/atom#22452] [Q&A] [answered]
I an using the material-UI theme, and because of an eye-disorder I want to make the (UI) text bigger in some places, for ex the text on the tabs, some menu options, text on buttons, etc. (NOT the text/code in the actual text editor. That I know how.).

For example, I want these menu options in a larger font-size:
<img width="202" alt="2BF24832-A31B-43A1-8C23-46C18FA84424" src="https://user-images.githubusercontent.com/70462421/118868645-b7c30e00-b8e4-11eb-9f15-3a6c821b49aa.png">

I’ve looked through both the atom’s general settings and the material theme’s settings, and I have scrolled through the material’s less-files but I don’t understand them (complete beginner + english is not first language…:-/)…

I have googled and searched Stack Overflow (and asked a question there, atm without an answer)..
https://stackoverflow.com/questions/67607570/change-ui-font-size-in-atom 
..but I can’t find what Im looking for. At least not explained for dummies :) Is it even possible? I know nothing.

I’m a complete newbie when it comes to code and text editors. Please keep in mind if answering.

Please help!

**Comment by ThatXliner** [comment]
Try [using some css](https://flight-manual.atom.io/using-atom/sections/basic-customization/#style-tweaks)

**Comment by Tr1ggah** [comment, marked as answer]
Use dev tools to find the element you want to change, then add it to your stylesheet
https://flight-manual.atom.io/using-atom/sections/basic-customization/
